CONTRIBUTING.md

```diff
@@ -6,11 +6,27 @@ When contributing to the `core-plans` repository, follow these guidelines.
 
 A package should contain key elements:
 
-- `pkg_name`
-- `pkg_license` (in [SPDX format](http://spdx.org/licenses/))
 - `pkg_description`
+- `pkg_license` (in [SPDX format](http://spdx.org/licenses/))
 - `pkg_maintainer` in the format of "The Habitat Maintainers <humans@habitat.sh>"
+- `pkg_name`
+- `pkg_origin`
+- `pkg_source`
 - `pkg_upstream_url`
+- `pkg_version`
+
+## Linting Your Plans
+
+It is good it use a tool to lint your plans to ensure you are not making any
+easy-to-catch mistakes in your scripting. We recommend running
+[ShellCheck](https://www.shellcheck.net/) against your plans.
+
+Some default checks of ShellCheck can be turned off, since they aren't
+necessarily applicable in plans. The following options work well:
+
+```
+shellcheck --shell=bash --exclude=SC1091,SC2034,SC2039,SC2148,SC2153,SC2154
```

Comment:
Is it worthwhile noting that you can run this lint + others with `git add <your changes> && pre-commit run`?